mysql-plugin/src/test/java/co/cask/mysql/MysqlPluginTestBase.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.mysql;
+
+import co.cask.DBRecord;
+import co.cask.DatabasePluginTestBase;
+import co.cask.cdap.api.plugin.PluginClass;
+import co.cask.cdap.api.plugin.PluginPropertyField;
+import co.cask.cdap.datapipeline.DataPipelineApp;
+import co.cask.cdap.proto.id.NamespaceId;
+import co.cask.db.batch.sink.ETLDBOutputFormat;
+import co.cask.db.batch.source.DataDrivenETLDBInputFormat;
+import com.google.common.base.Charsets;
+import com.google.common.base.Throwables;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Sets;
+import com.mysql.jdbc.Driver;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Map;
+import java.util.TimeZone;
+import javax.sql.rowset.serial.SerialBlob;
+
+public class MysqlPluginTestBase extends DatabasePluginTestBase {
+  protected static final String JDBC_DRIVER_NAME = "mysql";
+  protected static final String UI_NAME = "Mysql";
+
+  protected static String connectionUrl;
+  protected static Map<String, String> sinkProps;
+  protected static final int YEAR;
+  protected static boolean tearDown = true;
+
+  static {
+    Calendar calendar = Calendar.getInstance();
+    calendar.setTime(new Date(CURRENT_TS));
+    YEAR = calendar.get(Calendar.YEAR);
+  }
+
+  protected static final Map<String, String> BASE_PROPS = ImmutableMap.<String, String>builder()
+    .put("host", System.getProperty("mysql.host"))
```

Comment:
Can you add a few sentences to the README to describe how to run the unit tests with an example command

Reply:
I did few more changes to the tests structure but didn't change the logic. I added very brief explanation about how to run tests. Once more plugins are added I gonna think of better presentation of properties because list of -Doptions probably looks confusing.